Assets/Scripts/Inventory/Consumable.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Entity.Player;
+using Managers;
+using UnityEngine;
+using Inventory;
+
+[CreateAssetMenu(fileName = "New Consumable", menuName = "Inventory/Item/Consumable")]
+public class Consumable : Item
+{
+    [SerializeField]
+    private int healthValue;
+    [SerializeField]
+    private int saturationValue;
+    [SerializeField]
+    private int hydrationValue;
+
+    /// <summary>
+    /// Gets the PlayerState of the player, and changes its values depending on the item values.
+    /// </summary>
+    public override bool Use() 
+    {
+        PlayerState playerState = Singleton<PlayerManager>.Instance.GetPlayer().GetComponent<PlayerState>();
+        if (healthValue > 0)
+            playerState.Hit(-healthValue);
```

Comment:
you use the Hit method with a negative damage value to heal the player? 👀 😂 why not just use ChangePlayerHealth?

Reply:
Currently `ChangePlayerHealth` is private, so we would either have to change that or add another public method `void Heal(int amount)` or something like that

Reply:
or - if it needs more than the change of healthValue in the future (like triggering an animation or sound in some way) - implement a new medthod to playerState?

Reply:
> 
> 
> Currently `ChangePlayerHealth` is private, so we would either have to change that or add another public method `void Heal(int amount)` or something like that

as we had discussed already, yep, I like this 👍